### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [zidekuls/kaleidoscope#3] [Announcements]
# index

https://light.lionfree.net/

**Comment by zidekuls** [comment]
在摸鱼时间剖析了一下自己。我不是那种对人生有必胜信念的人，我的人生信条就是“差不多”，在工作和人生中，保证自己不掉队即可，奋勇拼搏和我没啥关系。
很明显的地方就是，我总是在即将掉队的时候捞自己一把，比如什么时候换工作，让自己涨点薪；什么时候加班输出一点东西，让老板看到我；定期存一点点钱，让自己稍微心安。
这种七十分的人生，真的很稳定也很持久，让我觉得是不是我已经默默牺牲掉了人生中的机会换来的，不过管他呢，奋斗多累。

**Comment by zidekuls** [comment]
法医报告：死亡教会我们什么

**Comment by zidekuls** [comment]
#3楼ll漓涳ll 福建省泉州市
我日常吃 维C 复合B族，葡萄糖酸锌，几块钱一瓶，一瓶可以吃一个多月。
#10988582
@2天前OO[3]XX[0]举报
